cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,55 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SetttingsObject from '../support/pages/settings.pageObject';
```

Comment:
There's a typo in the import statement. It should be `SettingsObject` instead of `SetttingsObject`. Please correct the class name in the import statement.